src/styles/variables.scss

```diff
@@ -22,9 +22,9 @@
 	--dswp-surface-color-primary-button-default: var(--bcds-surface-color-primary-button-default);
 	--dswp-banner-background-dark: #292929;
 	--dswp-theme-primary-gold: var(--bcds-theme-primary-gold);
-
+	--dswp-theme-hero-title-background: #013366b3;
```

Comment:
This color does exist in the BCDS tokens although without the transparency (--bcds-theme-primary-blue) so we could be using that here I think.